tests/tests.go

```diff
@@ -228,6 +232,33 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment := v1.Deployment{}
+	err := testClient.VirtClient.Get(context.TODO(), client.ObjectKey{Namespace: managerNamespace, Name: names.MANAGER_DEPLOYMENT}, &managerDeployment)
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get manager's Deployment")
+	Expect(managerDeployment.Spec.Template.Spec.Containers).ToNot(BeEmpty(), "Manager's deployment should contain containers")
+
+	for _, container := range managerDeployment.Spec.Template.Spec.Containers {
+		if container.Name != deploymentContainerName {
+			continue
+		}
+		r := regexp.MustCompile(fmt.Sprintf("--%s=(\\d+)", names.WAIT_TIME_ARG))
+		for _, arg := range container.Args {
+			match := r.FindStringSubmatch(arg)
+			if match == nil {
```

Comment:
Same here, we can change this `if` to 
```
if match != nil { 
.
.
.
}
```

Reply:
[Same](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/152#discussion_r429901532)